lib/winapi/xapi.c

```diff
@@ -5,9 +5,44 @@
  */
 
 #include <xapi.h>
+#include <assert.h>
 #include <stdio.h>
+#include <string.h>
 #include <xboxkrnl/xboxkrnl.h>
 
+__attribute__((constructor)) static VOID setup_d_drive (VOID)
```

Comment:
This should be using `void` instead of `VOID`.

Reply:
Are we sure we need to setup D: ourselves? The kernel should handle this based on the title path passed to the launch data page on reboot. Official XDK games don't handle setting up D themselves, but rely on the value set by the previously running Xbe (passed to the kernel via the launch data page when calling HalReturnToFirmware)

Reply:
Resolving this. If you want to continue discussing the D:/ situation, create a new discussion / review comment chain.